WORKSPACE

```diff
@@ -80,6 +80,14 @@ load(
 
 container_repositories()
 
+load("@io_bazel_rules_docker//repositories:deps.bzl", container_deps = "deps")
+
+container_deps()
+
+load("@io_bazel_rules_docker//repositories:pip_repositories.bzl", "pip_deps")
+
+pip_deps()
```

Comment:
Just curious, what does this part do?

Reply:
It loads the python dependencies that required by rules_docker. It's introduced in https://github.com/bazelbuild/rules_docker/pull/1534